src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -77,43 +81,48 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
         logger.quiet "Building image using context '${getInputDir()}'."
         def buildImageCmd
 
-        if(getDockerFile()) {
+        if (getDockerFile()) {
             logger.quiet "Using Dockerfile '${getDockerFile()}'"
             buildImageCmd = dockerClient.buildImageCmd()
                     .withBaseDirectory(getInputDir())
                     .withDockerfile(getDockerFile())
-        }
-        else {
+        } else {
             buildImageCmd = dockerClient.buildImageCmd(getInputDir())
         }
 
-        if(getTag()) {
+        if (getTag()) {
             logger.quiet "Using tag '${getTag()}' for image."
             buildImageCmd.withTag(getTag())
         }
 
-        if(getNoCache()) {
+        if (getNoCache()) {
             buildImageCmd.withNoCache(getNoCache())
         }
 
-        if(getRemove()) {
+        if (getRemove()) {
             buildImageCmd.withRemove(getRemove())
         }
 
-        if(getQuiet()) {
+        if (getQuiet()) {
             buildImageCmd.withQuiet(getQuiet())
         }
 
-        if(getPull()) {
+        if (getPull()) {
             buildImageCmd.withPull(getPull())
         }
 
-        if(getRegistryCredentials()) {
+        if (getRegistryCredentials()) {
             def authConfig = threadContextClassLoader.createAuthConfig(getRegistryCredentials())
             def authConfigurations = threadContextClassLoader.createAuthConfigurations([authConfig])
             buildImageCmd.withBuildAuthConfigs(authConfigurations)
         }
 
+        if (getBuildArgs()) {
```

Comment:
If we init to an empty map above we can skip the if-check here and just iterate over an empty map.